server/neptune/temporalworker/job/storage_backend.go

```diff
@@ -0,0 +1,180 @@
+package job
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/graymeta/stow"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+	"github.com/runatlantis/atlantis/server/logging"
+	"github.com/runatlantis/atlantis/server/neptune/logger"
+	"github.com/uber-go/tally/v4"
+)
+
+const OutputPrefix = "output"
+const PageSize = 100
+
+type StorageBackend interface {
+	Read(key string) ([]string, error)
+
+	// Activity context available
```

Comment:
I think what would help with this confusion, is just having things that are called within activities as part of the activities package and other things outside maybe?